include/aikido/perception/ObjectDatabase.hpp

```diff
@@ -44,7 +44,8 @@ class ObjectDatabase
   void getObjectByKey(
```

Comment:
Can we document this method? I'm not entirely clear what `obj_offset` is.

Reply:
I added the documentation for this. `objectName`, `objectResource`, and `objectOffset` are what should be retrieved from ObjectDatabase. For each key, those values are stored in `tag_data_xx.json` in `pr_ordata`.

Reply:
`objectOffset` is the offset between AprilTags and actual origin of objects.

Reply:
Should this be an offset or a transform? Since it's an `Isometry3d` it seems like it should be the latter?

Reply:
Some of them in the apriltag tag_data (https://github.com/personalrobotics/pr_ordata/blob/master/data/objects/tag_data_apriltags.json) have rotation components as well.